fcrepo/3.8.1-akubra/fedora-xacml-policies/repository-policies/default/deny-unallowed-file-resolution.xml

```diff
@@ -42,6 +42,6 @@
     </Condition>
   </Rule>
   -->
-  <Rule RuleId="1" Effect="Deny">
-  </Rule>
+  <!-- <Rule RuleId="1" Effect="Deny"> -->
```

Comment:
@jkremlacek Proč to máme zakomentované v Mulgaře? co to dělá?

Reply:
Řešeno mimo github... jde o konfiguraci Fedory pro ProArc, viz https://github.com/proarc/proarc/wiki/Instalace#fedora-commons